configure

```diff
@@ -12,7 +12,7 @@
 # Other settings: LTO, PGO?  Consider moving prefix, LTO, PGO to build and
 # install steps.
 
-TMP=${TMP:-/tmp}  # Assume that any system has $TMP set or /tmp exists
+TMP=${TMPDIR:-/tmp}  # Assume that any system has $TMP set or /tmp exists
```

Comment:
The comment is now out of date.  Does FreeBSD not have either TMP or /tmp ?

Reply:
Android does not have /tmp or $TMP, but it has $TMPDIR. FreeBSD has /tmp but not $TMP or $TMPDIr.